corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -253,7 +253,7 @@ <h4 class="modal-title">{% blocktrans with couch_user.human_friendly_name as fri
           </div>
           <div class="modal-footer">
             <a href="#" data-dismiss="modal" class="btn btn-default">{% trans 'Cancel' %}</a>
-            <button type="submit" class="btn btn-primary disable-on-submit">
+            <button type="submit" class="btn btn-primary disable-on-submit track-usage-link">
```

Comment:
Okay, it's not quite this easy. You also need to add data attributes with the category/action/label to record. Example: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templates/app_manager/partials/releases/releases_deploy_modal.html#L186-L194

Reply:
Oh! Ha sorry, I clearly didn't look at that carefully.

I'm curious what we do with this data. When I want to track things like this I usually use datadog (on the server side). Any reason to do it in GA in particular? (I haven't used GA much for years.) In this particular case would it solve a problem that could be solved in a pinch by grepping nginx logs? I think if I understand better when we do this, it'll help me make the call in the future

Reply:
I think if it's easy to look at usage of this request in datadog, that's fine. I was probably thinking about this as a button click and therefore thought of js analytics. We don't do anything methodical with that data I don't think, it's just there for querying.

Reply:
Okay, I think in the interests of just getting the feature out today, I'm going to skip this (and undo my ineffective change. In a pinch the way this feature's implemented, we can get all the information we need from the proxy access logs (since any usage will cause a POST to the force_user_412 url)